les/flowcontrol/control.go

```diff
@@ -316,6 +317,9 @@ func (node *ServerNode) CanSend(maxCost uint64) (time.Duration, float64) {
 	node.lock.RLock()
 	defer node.lock.RUnlock()
 
+	if node.params.BufLimit == 0 {
```

Comment:
So it's the root cause for overflowed stuff right? The bufferlimit is 0 but we don't have any protection.